tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -106,19 +122,38 @@ def _parse_arg_map(arg_map: str):
 
     return parsed_map
 
-def print_inputs_and_outputs_info(ie_network: IENetwork):
-    input_info = ie_network.input_info
-    for key in input_info.keys():
-        tensor_desc = input_info[key].tensor_desc
-        logger.info(f"Network input '{key}' precision {tensor_desc.precision}, "
-                                                    f"dimensions ({tensor_desc.layout}): "
-                                                    f"{' '.join(str(x) for x in tensor_desc.dims)}")
-    output_info = ie_network.outputs
-    for key in output_info.keys():
-        info = output_info[key]
-        logger.info(f"Network output '{key}' precision {info.precision}, "
-                                        f"dimensions ({info.layout}): "
-                                        f"{' '.join(str(x) for x in info.shape)}")
+
+def get_precision(element_type: Type):
```

Comment:
Can we use the same map for `get_element_type` and `get_precision`? These two functions should be symmetric

Reply:
No, we can't. It's symmetric, but reverse to each other.